_posts/2021-06-05-memn

```diff
@@ -0,0 +1,92 @@
+
+### Webbrowser:
+
+In Macbook gilt Safari als der eingebaute, schnelle Standard Webbrowser. Es gibt Dienste, wie das Intranet der ZHAW, Youtube TV welche nicht auf Safari funktionieren und andere Browser verwendet werden. Ich empfehle dir dabei diese zwei Webbrowser zu installieren.
+
+- **Google Chrome**: ist der beliebteste Browser und bietet durch seine vielen leistungsstarken Erweiterungen eine grosse Flexibilität und unterstützt zahlreiche Webanwendungen und Dienste.
+- **Mozilla Firefox**: konzentriert sich darauf, Tracker zu blockieren und datenschutzfreundliche Funktionen anzubieten. 
+
+### Arbeitsumgebung und Produktivität
+
+Der MacBook ist mit den iWork-Programmen Pages, Numbers und Keynote gut ausgestattet. Die Produktivität oder die Effizienz im Team steigern mittels folgenden Apps: 
+
+- **Adobe Acrobat Reader DC**: ermöglichst das Erstellen, Bearbeiten und Unterzeichnen von PDF-Dokumenten.
+- **Microsoft Office Education**: (vormals Office 365) ist eine Standardbürosoftware für das Studium, welche aus Excel (Tabellenkalkulation), OneNote (digitale Notizbücher), Outlook (E-Mail-Client mit Kalender und Aufgabenverwaltung), PowerPoint (Präsentation) Teams (Kommunikation), Visio (Diagramme), und Word (Textverarbeitung) besteht.
+- **OmniGraffle**: ist eine Möglichkeit die Organisation und Aufbereitung von Daten in übersichtliche Diagramme, Charts und Mindmaps zu illustrieren.
+- **Draw.io**: ist eine kostenlose Online-Diagrammsoftware zur Erstellung von Flussdiagrammen, Prozessdiagrammen, Organigrammen, und Netzwerkdiagrammen.
+
+### Kommunikation
+
+Für das Home Office und Remote Learning benutzen wir folgende Programme: 
+
+- **Cisco Webex Meetings**: ist ein Anbieter von Videokonferenzen, IP-Telefonie, Instant-Messaging, Dateiübertragung und Screen-Sharing.
+- **Zoom.us**: bietet Remote-Konferenzdienste, die Videokonferenzen, Online-Besprechungen, Chat und mobile Zusammenarbeit kombinieren
+
+### Entwicklungstools
+
+Damit du startklar für die Informatik-Kurse bist, brauchst du diese Programme:
+
+-**Git / GitHub**: ist ein freies verteiltes Versionskontrollsystem, um deine Arbeiten schnell und effizient zu bearbeiten. 
+-**Matlab**: ist eine Plattform für Programmierung und numerische Berechnungen zur Analyse von Daten, Entwicklung von Algorithmen und Erstellung von Modellen.
+-**R und RStudio**: ist eine Softwareumgebung für statistische Berechnungen und Grafiken. 
+- **Toggl Track**: ist eine Zeiterfassungssoftware und verfolgt Zeitblöcke, die optional mit einer Aufgabe, einem Projekt und Tags gekennzeichnet sind. 
+- **Xcode**: ist eine Entwicklungsumgebung um Programme für MacOs Benutzer zu entwicklen. 
+
+### Siehe auch
+
+Die [Studyhouse](https://www.studyhouse.ch/cgi-bin/home)können Software-Lizenzen für Studierende zu Sonderpreisen bezogen werden.
+
+### Literaturverzeichnis
```

Comment:
Verwende lieber Referenzen als Literaturverzeichnis. Das Meiste ist nämlich keine Literatur. Ich empfehle dir das auch in deinen anderen Dokumenten so beizubehalten.


```suggestion
### Referenzen
```